rocAL/include/loaders/image/node_cifar10_loader.h

```diff
@@ -48,5 +48,5 @@ class Cifar10LoaderNode : public Node {
     void update_node() override{};
 
    private:
-    std::shared_ptr<CIFAR10DataLoader> _loader_module = nullptr;
+    std::shared_ptr<CIFAR10Loader> _loader_module = nullptr;
```

Comment:
Why have we done the name change?

Reply:
To maintain parity with other loader names